shuup/simple_supplier/utils.py

```diff
@@ -36,23 +37,26 @@ def get_current_stock_value(supplier_id, product_id):
     events = (
         StockAdjustment.objects
         .filter(supplier_id=supplier_id, product_id=product_id)
+        .exclude(type=StockAdjustmentType.RESTOCK_LOGICAL)
         .aggregate(total=Sum("delta"))["total"] or 0)
     orders_bought = (
```

Comment:
strange name attribute as it has "nothing" to do with orders but mostly total quantity of bought products.